### DISCUSSION TITLE
My student pack was canceled, but I can't recertify

### DISCUSSION THREAD

**Discussion by yixijun** [community/community#64140] [GitHub Education]
### Hi everyone,

Hello, when I want to use the GitHub education pack, I find that it is not available, by checking the email I find that he is suspended and ask me to reverify, but when I return to the GitHub education page, it prompts me that there is an open issue and shows that I already have a students pack, as shown below

plz，tell me how to solve the problem，thanks 
!
![Screenshot_20230818_073815_com microsoft emmx_edit_284227678750900](https://github.com/community/community/assets/72327171/b3af7fbb-44a5-4b95-9806-33126056f752)

**Comment by bpeerless** [comment]


**Reply by yixijun** [reply]
thank you，i will try

**Reply by bpeerless** [reply]
If you end up getting anywhere, I would appreciate it if you marked this as the correct answer. Thank you!
